traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -313,6 +313,7 @@ func createV30(w http.ResponseWriter, r *http.Request, inf *api.APIInfo, reqDS t
 		&ds.RegionalGeoBlocking,
 		&ds.RemapText,
 		&ds.RoutingName,
+		&ds.ServiceCategoryId,
```

Comment:
this line should be moved to the end of the list to match where its at in your insertQuery()